testsuite/cli/src/client_proxy.rs

```diff
@@ -211,9 +215,22 @@ impl ClientProxy {
             sync_on_wallet_recovery,
             temp_files: vec![],
             quiet_wait,
+            url,
         })
     }
 
+    /// Gets a websocket client for the same node `DiemClient` connects to
+    pub async fn streaming_client(
+        &self,
+        config: Option<StreamingClientConfig>,
+    ) -> StreamResult<StreamingClient> {
+        let mut url = self.url.clone();
+        url.set_scheme("ws").expect("Could not set scheme");
+        url.set_path("/v1/stream/ws");
```

Comment:
are these constants somewhere?

Reply:
they're in the warp path from `/json-rpc/src/stream_rpc/transport/websocket.rs#L43`